modules/rubiconBidAdapter.js

```diff
@@ -329,11 +417,11 @@ function _getDigiTrustQueryParams() {
   if (!digiTrustId || (digiTrustId.privacy && digiTrustId.privacy.optout)) {
     return [];
   }
-  return [
-    'dt.id', digiTrustId.id,
-    'dt.keyv', digiTrustId.keyv,
-    'dt.pref', 0
-  ];
+  return {
+    'dt.id': digiTrustId.id,
+    'dt.keyv': digiTrustId.keyv,
+    'dt.pref': 0
+  };
```

Comment:
did this need to be changed?  was it not working previously?

Reply:
It was working previously, but everything was being stored in an array (opposed to plain object) to honor ordering of parameters, but since order does not matter I'm skipping the step of using an array, so I converted this to return an object for simpler key value assignment to the param object